pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -425,6 +428,7 @@ func RunServer(socketPath string,
 	grpcServer := grpc.NewServer([]grpc.ServerOption{}...)
 	server := &Launcher{
 		domainManager: domainManager,
+		agentData:     agentData,
```

Comment:
Nice catch.
I tried that as well, but came to a conclusion (need more opinions on that in case I am wrong),
that its not right to add a function to `DomainManager` interface, in order to access it, nor access it directly.
But I will need here 2nd opinion for sure.

Reasons:
1. Its not the scope in the scope of `DomainManager`.
2. The interface have mocking mechanism, usually its only for cases where we do need RPC / mock from what i understood from @vladikr (I might be wrong in this example).
3. Is it right (and possible) to access `agentData` directly without adding a function that will access it ?

Reply:
@AlonaKaplan wdyt about this heuristic question please?

Reply:
@oshoval , I do not understand your arguments here.
I see that the DomainManager has a `GetGuestInfo` already which is missing the interface. So what is the argument of not adding it there? Or if there is some objection for a good reason, to create a new method.

Reply:
Ye it has it, just wasn't sure if its right to add more there that, or to use the `AsyncAgentStore` directly (and not indirectly)
anyhow will do that